src/r4300/new_dynarec/assem_arm.c

```diff
@@ -2053,14 +2053,11 @@ static void emit_writebyte(int rt, int addr)
   output_w32(0xe5c00000|rd_rn_rm(rt,FP,0)|offset);
 }
 
-/*
-static void emit_mul(int rs)
+static void emit_mul(u_int rs1,u_int rs2,u_int rt)
 {
-  assem_debug("mul %%%s",regname[rs]);
-  assert(0);
+  assem_debug("mul %s,%s,%s",regname[rt],regname[rs1],regname[rs2]);
+  output_w32(0xe0000090|(rt<<16)|(rs2<<8)|rs1);
```

Comment:
It's the opcode of the MUL instruction ;-)

If you write this in assembly:
```mul r1,r2,r3 //r1 = r2*r3``` 

The compiler will generate the following binary code:
```0xe0010392```